test/realtimecontroller/DefaultRealtimeController.test.ts

```diff
@@ -538,6 +544,38 @@ describe('DefaultRealtimeController', () => {
       expect(callbackFired).to.be.true;
     });
 
+    it('will only unsubscribe from a given callback if provided', () => {
```

Comment:
Should probably have a test that verifies the passed callback is actually removed